sigs.yaml

```
@@ -82,6 +82,28 @@ sigs:
         - name: userinterface
           owners:
             - https://raw.githubusercontent.com/kubevirt/web-ui/master/OWNERS
+    - dir: kubevirt-sig-scale
+      name: KubeVirt Perf and Scale SIG
+      mission_statement: |
+        Define and drive scalability and performance goals within KubeVirt.  This includes documenting, testing, and measuring performance and scalability for a KubeVirt release.
+      label: kubevirt-community
+      leadership:
+        chairs: [rhallisey]
```

I think this could work according to https://github.com/kubevirt/community/blob/master/validators/cmd/sigs/sigs-validator.go#L54, didn't try it locally though

```suggestion
        chairs:
          - github: rthallisey
            name: Ryan Hallisey
            company : Nvidia
```